lib/sycamore/sycamore/query/operators/query_database.py

```diff
@@ -32,12 +32,8 @@ class QueryDatabase(Node):
                             }
                         }
                     },
-                    {
-                        "match": {
-                            "properties.path.keyword": "/path/to/data/*.pdf",
-                            "properties.entity.location": "Georgia"
-                        }
-                    }
+                    {"match": {"properties.entity.accidentNumber.keyword": "*K1234N*"}},
```

Comment:
Y U NO FORMAT

Reply:
Also the example text above needs to be updated if you are including the K1234N thing in the query.

Reply:
oh typo nice catch